<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SecondShifr</class>
 <widget class="QDialog" name="SecondShifr">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>820</width>
    <height>455</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Добро пожаловать в скрытый Шифр. Приложение по шифровке передаваемой и дешифровке получаемой информации</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true">border-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(9, 41, 4, 255), stop:0.085 rgba(2, 79, 0, 255), stop:0.19 rgba(50, 147, 22, 255), stop:0.275 rgba(236, 191, 49, 255), stop:0.39 rgba(243, 61, 34, 255), stop:0.555 rgba(135, 81, 60, 255), stop:0.667 rgba(121, 75, 255, 255), stop:0.825 rgba(164, 255, 244, 255), stop:0.885 rgba(104, 222, 71, 255), stop:1 rgba(93, 128, 0, 255));
position: absolute;</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>205</x>
     <y>0</y>
     <width>1500</width>
     <height>1500</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/recource/images/raduga1.jpg);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit_1">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>40</y>
     <width>210</width>
     <height>320</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit_2">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>40</y>
     <width>210</width>
     <height>320</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QLabel" name="label_1">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>10</y>
     <width>210</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI</family>
     <pointsize>12</pointsize>
     <weight>50</weight>
     <italic>true</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgba(127, 255, 0, 1);
color: rgba(255, 79, 0, 1);</string>
   </property>
   <property name="text">
    <string>Введите текст</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_3">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>745</x>
     <y>400</y>
     <width>71</width>
     <height>51</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="font">
    <font>
     <family>Goudy Stout</family>
     <pointsize>9</pointsize>
     <weight>75</weight>
     <italic>true</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
	float: right;
	margin-right: 3%;
	margin-top: 5px;
	border-radius: 5px;
	padding: 5px 9px;
	font-size: 1.2em;
	background-color: rgba(255, 36, 0, 1);
	border-bottom: 4px solid rgba(255, 36, 72, 1);
	color: white;
}

QPushButton:hover{
	background-color: rgba(255, 15, 0, 1);
	border-bottom: 4px solid rgba(217, 55, 60, 1);
}</string>
   </property>
   <property name="text">
    <string>Выход</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_About">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>750</x>
     <y>5</y>
     <width>61</width>
     <height>34</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
float: right;
margin-right: 3%;
margin-top: 5px;
border-radius: 5px;
padding: 5px 9px;
font-size: 1.2em;
background-color: rgba(63, 213, 75, 1);
border-bottom: 4px solid rgba(63, 213, 40, 1);
color: white;
}

QPushButton:hover
{
background-color: rgba(63, 230, 75, 1);
border-bottom: 4px solid rgba(63, 200, 5, 1);
}
</string>
   </property>
   <property name="text">
    <string>Qt</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>10</y>
     <width>210</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI</family>
     <pointsize>12</pointsize>
     <weight>50</weight>
     <italic>true</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgba(127, 255, 0, 1);
color: rgba(255, 79, 0, 1);</string>
   </property>
   <property name="text">
    <string>Полученный текст</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_4">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>50</y>
     <width>33</width>
     <height>33</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="font">
    <font>
     <family>Papyrus</family>
     <pointsize>14</pointsize>
     <weight>75</weight>
     <italic>false</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
float: right;
margin-right: 3%;
margin-top: 5px;
border-radius: 5px;
padding: 5px 9px;
font-size: 1.2em;
background-color: rgba(100, 149, 237, 1);
border-bottom: 4px solid rgba(0, 47, 140, 1);
color: white;
}

QPushButton:hover{
color:  white;
font-style: italic;
background-color: rgba(100, 165, 220, 1);;
}</string>
   </property>
   <property name="text">
    <string>?</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButton_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>135</y>
     <width>160</width>
     <height>35</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Vrinda</family>
     <pointsize>11</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QRadioButton{
background-color: rgba(100, 238, 238 ,1);
color: grey;
}
QRadioButton:hover{
background-color: rgba(50, 220, 255 ,1);
}</string>
   </property>
   <property name="text">
    <string>Текст в Биты</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButton_1">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>46</y>
     <width>160</width>
     <height>37</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe Print</family>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <italic>true</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QRadioButton {
background-color: rgba(255, 240, 0 ,1);
color: rgba(255, 79, 0, 1);
}
QRadioButton:hover {
background-color:  rgba(255, 250, 10 ,1);
}</string>
   </property>
   <property name="text">
    <string>Шифр Цезаря</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButton_4">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>244</y>
     <width>160</width>
     <height>46</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Century Gothic</family>
     <pointsize>11</pointsize>
     <weight>75</weight>
     <italic>false</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QRadioButton{
color: yellow;
background-color: rgba(255, 70, 192, 1);
}
QRadioButton:hover{
background-color: rgba(255, 70, 160, 1);
}</string>
   </property>
   <property name="text">
    <string>Шифр Виженера</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButton_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>184</y>
     <width>160</width>
     <height>46</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Vrinda</family>
     <pointsize>11</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QRadioButton{
background-color: black;
color: white;
}
QRadioButton:hover{
background-color: white;
color: black;
}</string>
   </property>
   <property name="text">
    <string>Биты в Текст</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>87</y>
     <width>84</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Perpetua Titling MT</family>
     <pointsize>9</pointsize>
     <weight>75</weight>
     <italic>true</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgba(255, 240, 0 ,1);
color: rgba(255, 79, 0, 1);</string>
   </property>
   <property name="text">
    <string>Введите шаг:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>107</x>
     <y>90</y>
     <width>74</width>
     <height>26</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Perpetua Titling MT</family>
     <pointsize>9</pointsize>
     <weight>75</weight>
     <italic>true</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit {
background-color: rgba(255, 240, 0 ,1);
color: rgba(255, 79, 0, 1);
}
QLineEdit:hover {
background-color: rgba(255, 250, 10 ,1);
}</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_5">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>130</y>
     <width>33</width>
     <height>33</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Papyrus</family>
     <pointsize>14</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
float: right;
margin-right: 3%;
margin-top: 5px;
border-radius: 5px;
padding: 5px 9px;
font-size: 1.2em;
background-color: rgba(100, 149, 237, 1);
border-bottom: 4px solid rgba(0, 47, 140, 1);
color: white;
}

QPushButton:hover{
color:  white;
font-style: italic;
background-color: rgba(100, 165, 220, 1);;
}</string>
   </property>
   <property name="text">
    <string>?</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_1">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>362</y>
     <width>103</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
float: right;
margin-right: 3%;
margin-top: 5px;
border-radius: 5px;
padding: 5px 9px;
font: DotumChe;
font-weight: bold;
font-size: 9pt;
background-color: rgba(255, 165, 0, 1);
color: white;
}

QPushButton:hover{
	/*background-color: rgba(40, 255, 1, 1);*/
color:  rgba(40, 255, 75, 1);
font: MS Reference Sans Serif;
font-size: 1.2em;
font-weight: bold;
font-style: italic;
background-color: rgba(255, 155, 0, 1);
}</string>
   </property>
   <property name="text">
    <string>Шифровка</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>383</x>
     <y>362</y>
     <width>110</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>MS Reference Sans Serif</family>
     <weight>75</weight>
     <italic>true</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
float: right;
margin-right: 3%;
margin-top: 5px;
border-radius: 5px;
padding: 5px 9px;
font-size: 1.2em;
background-color: rgba(255, 79, 0, 1);
color: white;
}

QPushButton:hover{
	background-color: rgba(242, 243, 244, 1);
color: black;
}</string>
   </property>
   <property name="text">
    <string>Дешифровка</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_6">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>190</y>
     <width>33</width>
     <height>33</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Papyrus</family>
     <pointsize>14</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
float: right;
margin-right: 3%;
margin-top: 5px;
border-radius: 5px;
padding: 5px 9px;
font-size: 1.2em;
background-color: rgba(100, 149, 237, 1);
border-bottom: 4px solid rgba(0, 47, 140, 1);
color: white;
}

QPushButton:hover{
color:  white;
font-style: italic;
background-color: rgba(100, 165, 220, 1);;
}</string>
   </property>
   <property name="text">
    <string>?</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_7">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>250</y>
     <width>33</width>
     <height>33</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Papyrus</family>
     <pointsize>14</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
float: right;
margin-right: 3%;
margin-top: 5px;
border-radius: 5px;
padding: 5px 9px;
font-size: 1.2em;
background-color: rgba(100, 149, 237, 1);
border-bottom: 4px solid rgba(0, 47, 140, 1);
color: white;
}

QPushButton:hover{
color:  white;
font-style: italic;
background-color: rgba(100, 165, 220, 1);;
}</string>
   </property>
   <property name="text">
    <string>?</string>
   </property>
  </widget>
  <widget class="Line" name="line_1">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>120</y>
     <width>160</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius: 1px;
background-color: rgba(63, 213, 75, 1);</string>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="Line" name="line_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>170</y>
     <width>160</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius: 1px;
background-color: rgba(63, 213, 75, 1);</string>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="Line" name="line_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>230</y>
     <width>160</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius: 1px;
background-color: rgba(63, 213, 75, 1);</string>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="Line" name="line_4">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>36</y>
     <width>160</width>
     <height>10</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius: 1px;
background-color: rgba(63, 213, 75, 1);</string>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="Line" name="line_5">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>340</y>
     <width>160</width>
     <height>10</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius: 1px;
background-color: rgba(63, 213, 75, 1);</string>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>36</y>
     <width>10</width>
     <height>314</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-bottom-left-radius: 5px;
background-color: rgba(63, 213, 75, 1);</string>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="Line" name="line_6">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>36</y>
     <width>10</width>
     <height>314</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-bottom-right-radius: 5px;
background-color: rgba(63, 213, 75, 1);</string>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>11</y>
     <width>180</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI</family>
     <pointsize>11</pointsize>
     <weight>75</weight>
     <italic>true</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border-top-left-radius: 10px;
border-top-right-radius: 10px;
background-color: rgba(127, 255, 0, 1);
color: rgba(255, 79, 0, 1);</string>
   </property>
   <property name="text">
    <string>Выберите Шифр</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>315</y>
     <width>160</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Perpetua Titling MT</family>
     <pointsize>9</pointsize>
     <weight>75</weight>
     <italic>false</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit {
background-color: rgba(255, 1, 255 ,1);
color: white;
}
QLineEdit:hover {
background-color: rgba(255, 70, 255 ,1);
}</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>290</y>
     <width>160</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI</family>
     <pointsize>9</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: white;
background-color: rgba(200, 0, 200, 1 );</string>
   </property>
   <property name="text">
    <string>Введите ключ:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QToolButton" name="toolButton">
   <property name="geometry">
    <rect>
     <x>-270</x>
     <y>210</y>
     <width>25</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>...</string>
   </property>
  </widget>
  <zorder>radioButton_2</zorder>
  <zorder>radioButton_1</zorder>
  <zorder>radioButton_4</zorder>
  <zorder>radioButton_3</zorder>
  <zorder>label_2</zorder>
  <zorder>lineEdit</zorder>
  <zorder>label</zorder>
  <zorder>textEdit_1</zorder>
  <zorder>textEdit_2</zorder>
  <zorder>pushButton_4</zorder>
  <zorder>label_1</zorder>
  <zorder>pushButton_3</zorder>
  <zorder>pushButton_About</zorder>
  <zorder>label_3</zorder>
  <zorder>pushButton_5</zorder>
  <zorder>pushButton_1</zorder>
  <zorder>pushButton_2</zorder>
  <zorder>pushButton_6</zorder>
  <zorder>pushButton_7</zorder>
  <zorder>line_1</zorder>
  <zorder>line_2</zorder>
  <zorder>line_3</zorder>
  <zorder>line_4</zorder>
  <zorder>line_5</zorder>
  <zorder>line</zorder>
  <zorder>line_6</zorder>
  <zorder>label_4</zorder>
  <zorder>lineEdit_2</zorder>
  <zorder>label_5</zorder>
  <zorder>toolButton</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
